core/src/main/java/org/jboss/projectmanipulator/core/ManipulationManager.java

```diff
@@ -38,13 +43,25 @@ public void init(ManipulationSession<R> session) throws ManipulationException {
     public void scanAndApply(final ManipulationSession<R> session) throws ManipulationException {
         // get project files list
         List<Project> projects = session.getProjects();
-        R result = session.getResult();
 
-        // apply manipulators on project files list and get changed ones back
-        Set<Project> changed = applyManipulations(projects, result);
+        Properties userProps = session.getUserProps();
+        boolean manipulationDisabled = false;
+        if (userProps.containsKey(MANIPULATION_DISABLE_PROPERTY)) {
+            String manipDisableValue = userProps.getProperty(MANIPULATION_DISABLE_PROPERTY);
+            manipulationDisabled = StringUtils.isEmpty(manipDisableValue) || "true".equalsIgnoreCase(manipDisableValue);
+        }
+
+        Set<Project> changed;
+        if (manipulationDisabled) {
```

Comment:
Shouldn't that be `!manipulationDisabled` ?

Reply:
It should, just found it too :-) Thanks!